get-started/quickstart.rst

```diff
@@ -0,0 +1,383 @@
+created conn.log has two entries. Here are those entries (some fields are cut
+for brevity with "..."):
+
+   .. code-block:: console
+
+
+     {
+        "ts": 1747147647.668533,
+        "uid": "CmgO6f3ddanrRjiCoc",
+        ...
+     },
+     {
+        "ts": 1747147654.27566,
+        "uid": "CQuwSY2FD0NsLn9CJj",
+        ...
+     }
+
+Now to find out more about both of these connections. Looking at the HTTP log,
+it also has two entries. Match these entries with the connection log using the
+timestamp and unique identifier (UID). The abbreviated logs for these two
+requests are:
+
+   .. code-block:: console
+
+     {
+         "ts": 1747147647.702181,
+         "uid": "CmgO6f3ddanrRjiCoc",
+         "method": "GET",
+         "host": "zeek.org",
+         "uri": "/",
+         ...
+     },
+     {
+         "ts": 1747147654.311012,
+         "uid": "CQuwSY2FD0NsLn9CJj",
+         "method": "WEIRD",
+         "host": "zeek.org",
+         "uri": "/",
+         ...
+     }
+
+The first entry is a simple ``GET`` request to zeek.org. The second entry has
+the same UID as the first entry in conn.log. The HTTP method was ``WEIRD``. For
+this case, Zeek will also generate a separate log for some unexpected behavior
+(or "weirds"), like an unknown HTTP method. You can find this in weird.log:
+
+   .. code-block:: console
+
+     {
+       "ts": 1747147654.311012,
+       "uid": "CQuwSY2FD0NsLn9CJj",
+       "name": "unknown_HTTP_method",
+       "addl": "WEIRD",
+       ...
+     }
+
+The UID for this entry is the same as the first entry in conn.log and the second
+entry in http.log. Therefore, there were two HTTP requests, one with a ``GET``
+request and one with a ``WEIRD`` request. The ``WEIRD`` request was rightfully
+classified as a “weird” by Zeek.
+
+More information on the various logs and what they report can be found in the
+:doc:`../logs/index` section. More information on working with logs can be found in
+the :doc:`../log-formats` section.
+
+Live Traffic
+============
+
+Zeek is often run on live traffic on a network, not just captured traffic.
+You can provide Zeek with a network interface to monitor traffic. Any traffic on
+that interface will be analyzed in order to create logs. For example, you may
+run Zeek on the ``en0`` network device (change ``en0`` to the device you want
+to monitor traffic on):
+
+.. code-block:: console
+
+     $ zeek -i en0
+
+Then, in another terminal, create the same two HTTP requests we saw earlier via
+``curl``:
+
+.. code-block:: console
+
+     $ curl -X GET http://zeek.org
+     $ curl -X WEIRD http://zeek.org
+
+Zeek will analyze traffic over that network device. Therefore, the logs may
+have more than just the two entries found before. Entries should still appear in
+:file:`conn.log`, :file:`http.log`, and :file:`weird.log` for these commands.
+
+Scripting
+=========
+
+Zeek can also use its own scripting language in order to configure behavior and
```

Comment:
```suggestion
You can also use Zeek's own scripting language in order to modify and extend its behavior:
```
If this is totally off from how you want this to sound, reject. Just seemed the user would be the one using the scripting language.